check-translations.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+// Adapted from: https://github.com/barryvdh/laravel-translation-manager/blob/master/src/Manager.php
+$translationKeys = [];
+$functions = ['@lang', '__'];
```

Comment:
Be aware that we could also run into a `@choice` and `trans_choice`
> https://laravel.com/docs/11.x/localization#pluralization

I don't think we currently use either of those functions in Rapidez

Reply:
Checked the list from the laravel-translation-manager config and added it here: https://github.com/rapidez/laravel-translation-checker/pull/2